### PR TITLE
Update hmacSignature header in Kosli integration

### PR DESCRIPTION
**Pull request by jumboduck**
**Requirements**

- [ ] I have followed the repository's [pull request submission guidelines](../blob/main/README.md#submitting-pull-requests)
- [ ] I have updated my user documentation
- [ ] I have updated my developer documentation (my integration README)
- [ ] I have submitted the [Become a Partner](https://launchdarkly.com/partners/) form

**Intended availability**

If you're adding a new integration, let us know if you are a community member or a partner.

If you are a partner, your integration will be available to all customers.

If you're part of the community and submitting an integration for your own purposes or internal teams, specify your company name so that we can verify your identity and enable the integration for your account.

**Related issues**

Provide links to any issues in this repository or elsewhere relating to this pull request.

**Describe the solution you've provided**

If you're adding a new integration, provide some context into how LaunchDarkly and the third-party service should interact.

If you're modifying an existing integration, provide a clear and concise description of what you expect to happen.

**Describe alternatives you've considered**

If you're modifying an existing integration, provide a clear and concise description of any alternative solutions or features you've considered.

**Provide a request preview**

Include the output from running `npm run preview <INTEGRATION_NAME>`.

**Additional context**

Add any other context about the pull request here.

### CODE REVIEW

File: integrations/kosli/manifest.json [hunk 1]
```diff
@@ -38,10 +38,14 @@
...
-            "value": "{{secret}}"
+            "name": "Content-Type",
+            "value": "application/json"
           }
-        ]
+        ],
+        "hmacSignature": {
+          "headerName": "X-LD-Signature",
+          "hmacSecretFormVariableKey": "secret"
+        }
```
Comment (monsagri) [inline integrations/kosli/manifest.json:48]: Is the intent here to switch to a different form of authentication for the integration? 
If that is the case, and you would want to start signing messages with an HMAC signature, further code changes would be required

File: integrations/kosli/manifest.json [hunk 2]
```diff
@@ -38,10 +38,14 @@
...
-            "name": "x-ld-signature",
-            "value": "{{secret}}"
+            "name": "Content-Type",
+            "value": "application/json"
           }
-        ]
+        ],
+        "hmacSignature": {
+          "headerName": "X-LD-Signature",
+          "hmacSecretFormVariableKey": "secret"
```
Comment (monsagri) [inline integrations/kosli/manifest.json:47]: The `{{secret}}` syntax in the existing header is Handlebars syntax in order to use the value of the secret formVariable defined on line 27. 

If the plan is to only refer to the key and process the formVariable value elsewhere, could you please provide more detail on how you are planning to achieve that?

**Review by monsagri — CHANGES_REQUESTED** [review]
Hi @jumboduck!

Thanks for raising this PR, could you please provide some more detail on what the intended effect of the change is, ideally by filling out the PR request template?

File: integrations/kosli/manifest.json (hunk 2 above)
Comment (sloloris) [reply integrations/kosli/manifest.json:47]: i told @InTheCloudDan i think we do it this way! i looked at the other integrations using hmac config via manifest (`compass` and `msteams`) and i think counterintuitively we actually want the field name in this field as opposed to the field value

Comment (monsagri) [reply integrations/kosli/manifest.json:47]: Makes sense, just realised we're switching over to the inbuilt HMAC now

File: integrations/kosli/manifest.json (hunk 1 above)
Comment (jumboduck) [reply integrations/kosli/manifest.json:48]: Yes the intention was always to use hmac validation. The manifest was not implemented correctly, this is a fix.

File: integrations/kosli/manifest.json (hunk 2 above)
Comment (jumboduck) [reply integrations/kosli/manifest.json:47]: @monsagri we indeed copied what we saw compass do in their integration

Is there anything else you need on our end?

**Review by InTheCloudDan — APPROVED** [review]

**Review by monsagri — APPROVED** [review]
sorry about the delay on this, didn't see the re-request

**Pull request merged**